### PR TITLE
Fix/aut 2967/upper bound field value

### PR DESCRIPTION
**Pull request by Karol-Stelmaczonek**
Add isNaN() checks for value set in Min and Max fields. Now you can't use delete key on value and save empty min max input.

Here's the link to [task](https://oat-sa.atlassian.net/browse/AUT-2967)  In it we have description how to test it

### CODE REVIEW

File: views/js/qtiCreator/widgets/component/minMax/minMax.js [hunk 1]
```diff
@@ -360,6 +360,13 @@ define([
 
                     fromField = fromField || fields.min;
 
+                    if (isNaN(this.getMinValue())) {
+                        this.setMinValue(config.lowerThreshold);
+                    }
+                    if (isNaN(this.getMaxValue())) {
+                        this.setMaxValue(config.lowerThreshold);
```
Comment (jsconan) [inline views/js/qtiCreator/widgets/component/minMax/minMax.js:367]: **question:** Why is the fallback set with the lower threshold? Shouldn't it be the `upperThreshold` instead?

**suggestion:**
```suggestion
                        this.setMaxValue(config.upperThreshold);
```

**Review by jsconan — COMMENTED** [review]
The code looks straightforward.
However, I wonder if the logic is accurate. Shouldn't the default for the max value be the upper threshold?

Besides this, I'd encourage you to give more info in your PR description. It is a bit terse.
The good practice is to:
- give a link to the Jira ticket
- explain the purpose
- explain how to test

Please keep in mind a PR is part of the history, and the more info you give, the better it is for understanding the intent and the context several months later...

File: views/js/qtiCreator/widgets/component/minMax/minMax.js (hunk 1 above)
Comment (Karol-Stelmaczonek) [reply views/js/qtiCreator/widgets/component/minMax/minMax.js:367]: I was thinking about it but mostly people put there low numbers and after this operation (when user deletes value from input) input will be filed wit max int value. I don't think its super user friendly and intuitive. Sure but still we can change it but I don't like it.

**Review by jsconan — APPROVED** [review]
It works well. The zlowerBoundz and `upperBound` attributes are set to the default value when deleted.
image

Any chance to see a unit test for the case?

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] ~~New code is not subject to concurrency issues (if applicable)~~
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic

**Review by marpesia — APPROVED** [review]
Good solution 👍🏻 

**Review Checklist**
- [x] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
- [ ] Pull request's target is not `master`
- [ ] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] Commits messages are meaningful
- [ ] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Pull request merged**